src/components/Avatar/Avatar.jsx

```diff
@@ -139,11 +165,20 @@ const Avatar = ({
           {...overrideTooltipProps}
         >
           <div
-            className={cx(bemHelper({ element: "circle" }), bemHelper({ element: "circle", state: type }), {
-              [bemHelper({ element: "circle", state: "is-disabled" })]: overrideDisabled,
-              [bemHelper({ element: "circle", state: "is-square" })]: overrideSquare,
-              [bemHelper({ element: "circle", state: "without-border" })]: withoutBorder
-            })}
+            className={cx(
+              styles.circle,
+              "monday-style-avatar_circle",
+              styles[`${camelCase("circle--" + type)}`],
+              `monday-style-avatar_circle--${type}`,
+              {
+                [styles.circleIsDisabled]: overrideDisabled,
```

Comment:
please switch naming to disabled, square, withoutBorder.... (the circle prefix not needed and create weird names such as circleIsSquare)